_champions/ghoneycutt.md

```diff
@@ -3,6 +3,11 @@ layout: profile
 name: Garrett Honeycutt
 image: https://avatars3.githubusercontent.com/u/313891?v=4
 bio: Providing Puppet automation consulting and training at Tailored Automation
+twitter: _ghoneycutt
+github: ghoneycutt
+linkedin: garretthoneycutt
```

Comment:
@binford2k is this right for LinkedIn? The url is `https://www.linkedin.com/in/garretthoneycutt/`

Reply:
yep!